src/integration/cloneDocuSources.js

```diff
@@ -15,7 +18,6 @@ const eachRegTopic = require('../helpers/registryIterator'),
  * @param {Function} [next] - callback for asynch operations
  */
 function cloneDocuSources(registry, config, next) {
```

Comment:
now once we have implementation you can refactor this js and write it properly, do not mix logic in `cloneDocuRepo` and create new function copyDocu and in the iterateRegClone function do the if(topicDetails.local)